doc/parameters/param_list.md

```diff
@@ -597,8 +597,7 @@ When `recovery_plugins` parameter is not overridden, the following default plugi
 
 | Input Port | Default | Description |
 | ---------- | ------- | ----------- |
-| position | N/A | Position |
-| orientation | N/A | Orientation |
+| goal | N/A | Goal |
```

Comment:
This will need to be updated in the navigation.ros.org repo's param files & also to the migration guide

Reply:
- `the param files` is https://navigation.ros.org/configuration/packages/bt-plugins/actions/NavigateToPose.html ?
- `the migration guide` is https://navigation.ros.org/migration/Eloquent.html# ?

Reply:
This would be going into G-turtle not Foxy, so a new file for Foxy->Galactic (if doesn't already exist)

Otherwise, yes

Reply:
I've created https://github.com/ros-planning/navigation.ros.org/pull/49.